htmdx/cli.py

```diff
@@ -73,25 +74,15 @@ def check_registration(product=None):
     if not product:
         product = "NA"
 
-    prefix = os.path.join(os.path.expanduser('~'), '.htmd')
-    if not os.path.exists(prefix):
-        os.mkdir(prefix)
-    prefix = os.path.join(prefix, '.registered-' + product)
+    reg_file = os.path.join(os.path.expanduser('~'), '.htmd', '.registered-htmd', 'registration')
 
-    if not os.path.exists(prefix):
-        if not check_approval(product, prefix):
-            if os.getenv("LICENCE_ACCEPTED") == "YES" or os.getenv("TRAVIS_REPO_SLUG"):
-                print("Licence accepted automatically. License terms apply")
-                return 
-            else:
-                if product == 'htmd':
-                    print('\nHTMD License accepted automatically. Check license here: '
-                          'https://raw.githubusercontent.com/Acellera/htmd/master/htmd/LICENCE.txt')
-                    print('\nFor advanced features (e.g. parameterize) and to remove this message, we recommend '
-                          'registering. Run htmd_register in your terminal.')
-                else:
-                    accept_license(product=product)
-                    do_register(product=product)
+    if os.path.isfile(reg_file):
+        if not check_approval(product, reg_file):
```

Comment:
```
if not (os.path.isfile(reg_file) and check_approval(product, reg_file)):
    accept_license(product=product)
    do_register(product=product)
```
More elegant, no code duplication. Ifs short-circuit anyway so there is no need to do nested ifs like that